src/components/button/index.js

```diff
@@ -127,6 +126,14 @@ export default class Button extends BaseComponent {
     }
   }
 
+  // This method will be called more than once in case of layout change!
+  getComponentDimensions(event) {
+    if (Platform.OS === 'android' && Platform.Version <= 17) {
+      const height = event.nativeEvent.layout.height;
```

Comment:
Use destructor for extracting the height as follow
const {height} = event.nativeEvent.layout;